modules/ROOT/pages/upgrade-index.adoc

```diff
@@ -1,35 +1,278 @@
+
+.. Plan to upgrade when your team is not deploying new applications or managing existing applications on Runtime Fabric.  
+.. Navigate to your Runtime Fabric by selecting it on the *Runtime Fabrics* tab in Runtime Manager.
+.. Select "Upgrade to vx.x". The status of the Runtime Fabric is updated to reflect the upgrade procedure is in progress.
+.. Wait for the status to transition to Active. 
++
+In most cases, the upgrade takes fewer than 10 minutes. When the status transitions to Active, and the version of the Runtime Fabric reflects the new version, the upgrade is complete.
++
+If the Runtime Fabric and Anypoint Runtime Manager connection disconnects during the upgrade, connectivity is restored after the upgrade finishes.
++
+Keep your browser session open to use in later steps.
++
+[IMPORTANT]
+To upgrade to Runtime Fabric version 1.5 or later, you must be running version 1.4.54. If you are not running version 1.4.54, upgrade to 1.4.54 before upgrading to 1.5.
+. If you are not using the latest version of the `rtfctl` command-line utility, download the latest version:
+.. Using your terminal open on a controller node, run the following command: 
++
+----
+cd /opt/anypoint/runtimefabric
+sudo curl -L https://anypoint.mulesoft.com/runtimefabric/api/download/rtfctl/latest -o rtfctl
+----
++
+.. Change file permissions for the `rtfctl` binary: 
++
+----
+sudo chmod +x rtfctl
+----
+. If you are not using the latest version of the cluster-level components (installer package), upgrade to the latest version:
+
+.. Make sure the `/opt/anypoint/runtimefabric/installer` directory has a minimum of 4 GiB of disk space available.
+.. Verify you have root privileges. Root privileges are required to perform an upgrade.
+.. Verify your environment meets the prerequisites:
 
-If an appliance upgrade is available, a code snippet for performing the upgrade is provided, as shown in the following example:
+*** A production configuration of Runtime Fabric, with a minimum of 3 controller VMs, is required.
+*** The internal load balancer must run on at least 3 replicas to maintain availability.
+*** An external load balancer must be configured to load balance incoming requests to the controller VMs, with a health check configured for TCP port 443.
+*** At least one worker node’s worth of CPU and memory resources must be available in the cluster.
++
+This enables safe removal of a worker node from the cluster to apply upgrades, without impacting availability of applications.
+*** Applications serving inbound requests must be scaled to a minimum of 2 replicas.
++
+[WARNING] Runtime Fabrics running on cluster version 1.0.x encounter downtime to applications if configured to use an HTTP proxy. You must perform an additional step in the procedure by applying the HTTP proxy configuration on each node after the upgrade. Locate the Runtime Fabric version by running `gravity status` on a node and referencing the cluster version.
```

Comment:
Correction: gravity status -> rtfctl version

"Locate the Runtime Fabric version by running `rtfctl version` on a node and referencing the appliance version."

Reply:
Good catch - thank you!